From aa81ee8c23010c81dc85d2f7dd33ab03220e6a2d Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Fri, 17 Nov 2023 08:51:55 +0100
Subject: [PATCH 0650/1141] arm64: dts: st: enable the i2c2 node in
 stm32mp257f-dk board

Enable the i2c2 node for stm32mp257f-dk board

Signed-off-by: Olivier BIDEAU <olivier.bideau@st.com>
Signed-off-by: Patrice Chotard <patrice.chotard@foss.st.com>
Change-Id: Ie9520b0906d7896c1929d743c837e6ded39c79c4
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
index a2e46a364a3a..0be9f900b98d 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -94,6 +94,19 @@ phy1_eth1: ethernet-phy@1 {
 	};
 };
 
+&i2c2 {
+	pinctrl-names = "default", "sleep";
+	pinctrl-0 = <&i2c2_pins_b>;
+	pinctrl-1 = <&i2c2_sleep_pins_b>;
+	i2c-scl-rising-time-ns = <185>;
+	i2c-scl-falling-time-ns = <20>;
+	clock-frequency = <100000>;
+	status = "okay";
+	/* spare dmas for other usage */
+	/delete-property/dmas;
+	/delete-property/dma-names;
+};
+
 &mlahb {
 	intc_rpmsg: interrupt-controller@1 {
 		compatible = "rpmsg,intc";
-- 
2.39.2

